code/src/stf/include/interface/stf/cam/frustum.hpp

```diff
@@ -85,20 +93,72 @@ namespace stf::cam
          */
         bool intersects_fast(aabb_t const& aabb) const
         {
-            for (size_t i = 0; i < c_num_planes; ++i)
+            if (m_aabb.intersects(aabb))
```

Comment:
The meat of this PR is in this method. We first add an easy bounding box check in hopes of avoiding more expensive tests. Then check box vertices against frustum planes. Finally we check frustum vertices against box plane. If we find that any plane entirely excludes the other shape, then we know that shapes do not intersect. Otherwise, we just return true